src/governance/object.h

```diff
@@ -30,10 +30,10 @@ extern RecursiveMutex cs_main;
 static constexpr double GOVERNANCE_FILTER_FP_RATE = 0.001;
 
 
-static constexpr CAmount GOVERNANCE_PROPOSAL_FEE_TX = (1 * COIN);
+static constexpr CAmount GOVERNANCE_COMMITMENT_AMOUNT = (1 * COIN);
```

Comment:
@UdjinM6, can you explain the rationale/purpose for the 1 and 6 block confirmations here? Does it make more sense now to require InstantSend and/or ChainLocks for proposal validity instead of block confirmations (in a separate issue/PR)?